packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -292,15 +300,11 @@ export type TableProps<
 } & Omit<CommonProps, 'title'>;
 
 // Original type for some reason is missing sub-columns
```

Comment:
I think this comment was for `ColumnType` which has been removed. Other than that, things look good to me!

Reply:
Removed